test/integration/file-uploads/request-body.test.js

```diff
@@ -651,26 +671,16 @@ describe('file upload in request body', function () {
         });
     });
 
-    (IS_BROWSER ? describe.skip : describe)('should upload large files correctly', function () {
-        var testUploadFile = 'test/fixtures/upload-file-large.json';
-
+    (IS_BROWSER ? describe.skip : describe)('large file upload in request body', function () {
         afterEach(function () {
```

Comment:
We're deleting the file in `afterEach` whereas creating it in `before`. If another test is added in this `describe` block, it will fail because the file will not be available.

Let's either use `beforeEach` and `afterEach` or use `before` and `after`.